app/classifier/tasks/dropdown/dropdown-dialog.cjsx

```diff
@@ -58,22 +66,29 @@ DropdownDialog = React.createClass
     select.allowCreate = @refs.allowCreate.checked
     @setState editSelect: select
 
-  onChangeConditionalAnswer: (select, e) ->
+  onChangeConditionalAnswer: (select, index, e) ->
     @handleOptionsKeys(select, e.target.value)
-    for select in @props.related
-      @handleOptionsKeys(select, null)
+    if @state.conditionalSelects.length and index isnt (@state.conditionalSelects.length - 1)
+      for otherIndex in [(index + 1)..(@state.conditionalSelects.length - 1)]
+        @handleOptionsKeys(@state.conditionalSelects[otherIndex], null)
 
   addOption: (optionLabel) ->
+    addOption: (optionLabel) ->
     if not optionLabel
-      return window.alert('Please provide an option')
+      alertCount++
+      return window.alert('Please provide an option.')
 
     if @getOptionsByProp("label")?.indexOf(optionLabel) isnt -1
-      return window.alert('Options must be unique within each dropdown')
+      alertCount++
+      return window.alert('Options must be unique within each dropdown.')
 
     select = @state.editSelect
 
     if select.condition? and not @state.optionsKeys[select.condition]
```

Comment:
should delete now that adding options disabled until conditional options selected?